### PR TITLE
renames "Mailing List" to "Newsletter" across languages

### LINKED ISSUE #264: Change "Mailing List" to "Newsletter" on the website
"Newsletter" sounds nicer.

### PR DESCRIPTION
**Pull request by andrea-kyurchiev**
Translations are made with Google translate so might be useful to have a native speaker look over them before merge. 

Closes https://github.com/InnerSourceCommons/InnerSourceMarketing/issues/264

### CODE REVIEW

**Comment by rrrutledge** [comment]
How did you do the translations?

File: content/pt-br/_index.md [hunk 1]
```diff
@@ -96,8 +96,8 @@ title: "InnerSource Commons"
...
-        <p class="h3 section-title">Cadastre-se em nossa lista de e-mails</p>
+            <p class="mb-4">A maior parte das ações na comunidade acontece em nosso <a href="https://join.slack.com/t/innersourcecommons/shared_invite/zt-1msf8vcqu-fYEHcyI1l4eSPq6rGprMXA">Canal do Slack</a>. Você também pode se manter informado sobre as novidades, eventos e atividades da InnerSource Commons assinando nossa newsletter abaixo, nos seguindo no <a href="https://twitter.com/InnerSourceOrg">Twitter</a> e no <a href="https://www.linkedin.com/company/innersourcecommons">LinkedIn</a>, e se inscrevendo em nosso <a href="https://www.youtube.com/channel/UCoSPSd6Or4F_vpjo4SmyoEA">Canal do YouTube</a>.</p>
+        <p class="h3 section-title">Junte-se ao nosso boletim informativo</p>
```
Comment (fer-correa) [inline content/pt-br/_index.md]: Cadastre-se em nossa newsletter

File: content/pt-br/about/contact.md [hunk 2]
```diff
@@ -29,8 +29,8 @@ type: wide
...
-        <p class="h3 section-title">Inscreva-se em nossa lista de e-mails</p>
+        <p class="mb-4">A maioria das atividades da comunidade acontece em nosso <a href="https://join.slack.com/t/innersourcecommons/shared_invite/zt-1msf8vcqu-fYEHcyI1l4eSPq6rGprMXA">canal do Slack</a>. Você também pode se manter informado sobre as novidades, eventos e atividades da InnerSource Commons assinando nossa newsletter abaixo, nos seguindo no <a href="https://twitter.com/InnerSourceOrg">Twitter</a> e no <a href="https://www.linkedin.com/company/innersourcecommons">LinkedIn</a>, e se inscrevendo em nosso <a href="https://www.youtube.com/channel/UCoSPSd6Or4F_vpjo4SmyoEA">canal do YouTube</a>.</p>
+        <p class="h3 section-title">Junte-se ao nosso boletim informativo</p>
```
Comment (fer-correa) [inline content/pt-br/about/contact.md]: Cadastre-se em nossa newsletter

File: content/pt-br/about/privacy.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ Esta política de privacidade explicará como nossa organização utiliza os dad
 
 A InnerSource Commons coleta a maioria dos dados diretamente fornecidos por você. Coletamos e processamos dados quando você:
 
-- Se inscreve em nossa lista de e-mails: A InnerSource Commons coleta informações de identificação pessoal (nome e endereço de e-mail) para enviar por e-mail notícias, eventos e artigos que achamos que você pode gostar.
+- Se inscreve em nossa newsletter: A InnerSource Commons coleta informações de identificação pessoal (nome e endereço de e-mail) para enviar por e-mail notícias, eventos e artigos que achamos que você pode gostar.
```
Comment (fer-correa) [inline content/pt-br/about/privacy.md]: - Cadastre-se em nossa newsletter: A InnerSource Commons coleta informações de identificação pessoal (nome e endereço de e-mail) para enviar por e-mail notícias, eventos e artigos que achamos que você pode gostar.

File: content/pt-br/community/_index.md [hunk 4]
```diff
@@ -182,8 +182,8 @@ as as discussões em nosso canal Slack seguem a <a href="https://www.chathamhous
...
-        <p class="h3 section-title">Inscreva-se em nossa lista de e-mails</p>
+        <p class="mb-4">A maioria das atividades da comunidade acontece no nosso <a href="https://join.slack.com/t/innersourcecommons/shared_invite/zt-1msf8vcqu-fYEHcyI1l4eSPq6rGprMXA">Canal Slack</a>. Você também pode se manter informado sobre as notícias, eventos e atividades da InnerSource Commons assinando nossa newsletter abaixo, nos seguindo no <a href="https://twitter.com/InnerSourceOrg">Twitter</a> e <a href="https://www.linkedin.com/company/innersourcecommons">LinkedIn</a>, e se inscrevendo em nosso <a href="https://www.youtube.com/channel/UCoSPSd6Or4F_vpjo4SmyoEA">Canal no YouTube</a>.</p>
+        <p class="h3 section-title">Junte-se ao nosso boletim informativo</p>
```
Comment (fer-correa) [inline content/pt-br/community/_index.md]: Cadastre-se em nossa newsletter

**Review by fer-correa — CHANGES_REQUESTED** [review]
Hi all,
I added some small recommendations for adjustments to the text of the translation into Brazilian Portuguese.

File: content/ja/community/_index.md [hunk 5]
```diff
@@ -161,7 +161,7 @@ title: "コミュニティ"
         <p class="h2 section-title">最新情報はこちら</p>
         <p class="mb-4">
         コミュニティ活動のほとんどは、<a href="https://innersourcecommons.org/slack//">Slack</a> で行われています。また、メールの定期購読や、 <a href="https://twitter.com/InnerSourceOrg">Twitter</a>、 <a href="https://www.linkedin.com/company/innersourcecommons">LinkedIn</a> のフォロー、<a href="https://www.youtube.com/channel/UCoSPSd6Or4F_vpjo4SmyoEA">YouTube</a>チャンネルの登録やによって、InnerSource Commons のニュース、イベントについての情報を取得することができます。</p>
-        <p class="h3 section-title">メールを定期購読する</p>
+        <p class="h3 section-title">ニュースレターに参加する</p>
```
Comment (yuhattor) [inline content/ja/community/_index.md]:
```suggestion
        <p class="h3 section-title">ニュースレターを定期購読する</p>
```

File: content/ja/about/_index.md [hunk 6]
```diff
@@ -97,7 +97,7 @@ title: "InnerSource Commons"
         <p class="h2 section-title">最新情報はこちら</p>
         <p class="mb-4">
         コミュニティ活動のほとんどは、<a href="https://innersourcecommons.org/slack//">Slack</a> で行われています。また、メールの定期購読や、 <a href="https://twitter.com/InnerSourceOrg">Twitter</a>、 <a href="https://www.linkedin.com/company/innersourcecommons">LinkedIn</a> のフォロー、<a href="https://www.youtube.com/channel/UCoSPSd6Or4F_vpjo4SmyoEA">YouTube</a>チャンネルの登録やによって、InnerSource Commons のニュース、イベントについての情報を取得することができます。</p>
-        <p class="h3 section-title">メールを定期購読する</p>
+        <p class="h3 section-title">ニュースレターに参加する</p>
```
Comment (yuhattor) [inline content/ja/about/_index.md]:
```suggestion
        <p class="h3 section-title">ニュースレターを定期購読する</p>
```

File: content/ja/_index.md [hunk 7]
```diff
@@ -97,7 +97,7 @@ title: "InnerSource Commons"
         <p class="h2 section-title">最新情報はこちら</p>
         <p class="mb-4">
         コミュニティ活動のほとんどは、<a href="https://innersourcecommons.org/slack/">Slack</a> で行われています。また、メールの定期購読や、 <a href="https://twitter.com/InnerSourceOrg">Twitter</a>、 <a href="https://www.linkedin.com/company/innersourcecommons">LinkedIn</a> のフォロー、<a href="https://www.youtube.com/channel/UCoSPSd6Or4F_vpjo4SmyoEA">YouTube</a>チャンネルの登録やによって、InnerSource Commons のニュース、イベントについての情報を取得することができます。</p>
-        <p class="h3 section-title">メールを定期購読する</p>
+        <p class="h3 section-title">ニュースレターに参加する</p>
```
Comment (yuhattor) [inline content/ja/_index.md]:
```suggestion
        <p class="h3 section-title">ニュースレターを定期購読する</p>
```

**Comment by yuhattor** [comment]
🇯🇵 Done 🙌

**Comment by rrrutledge** [comment]
Thanks for all of these reviews!  @andrea-kyurchiev is out this week but should be able to take a look next week.

**Review by CiaraFlanagan — APPROVED** [review]

**Comment by CiaraFlanagan** [comment]
Hi @rrrutledge, @andrea-kyurchiev

I've merged the proposed change from 'mailing list' to 'newsletter'. Will keep my eye on this to see if it looks ok on deployment.

**Pull request merged**